provisioning/roles/docker/files/docker-compose.yml

```diff
@@ -38,6 +38,10 @@ services:
       - "5601:5601"
     depends_on:
       - elasticsearch
+    logging:
+      options:
+        max-size: "1M"
```

Comment:
Why are some configured with `max-size: "1M"` and others with `max-size: "10M"`?

Reply:
I thought not all logs are important but I can make them equal to be safe.